lib/ViewModels/TerriaViewer.js

```diff
@@ -338,6 +338,16 @@ us via email at '+that.terria.supportEmail+'.'
 
     scene.globe.depthTestAgainstTerrain = false;
 
+    // if both terriajs default credit and cecium ion credits are required
+    if(defined(this._defaultTerriaCredit) && defined(this._bumpyTerrainCredit)){
+      // This line worked for sometimes, then stopped working. Therefore I also use CSS to hide it in case it doesn't work
+      scene.frameState.creditDisplay.removeDefaultCredit(this._bumpyTerrainCredit);
+      const combinedCredit = new Credit(`<div id='logo-credit'>${this._defaultTerriaCredit._html} ${this._bumpyTerrainCredit._html}</div>`);
```

Comment:
I think you can add multiple default credits, instead of creating a combined one. Or did that not work?

Reply:
it is to make them display in the layout we want

Reply:
Is it possible to use CSS to achieve the layout we want, instead of combining the credits? I think we already have some logic to treat the first credit as special, so it would just be a matter of treating the first two specially, right?